Python/bltinmodule.c

```diff
@@ -3016,7 +3016,14 @@ static PyMethodDef builtin_methods[] = {
 PyDoc_STRVAR(builtin_doc,
 "Built-in functions, exceptions, and other objects.\n\
```

Comment:
```suggestion
"Built-in functions, types, exceptions, and other objects.\n\
```
This is subjective, but I think the built-in types are among the most common ones where you need to use the builtins module directly (for example, having a `set` function while also wanting to use `builtins.set`).